cfgov/v1/views/reports.py

```diff
@@ -17,6 +19,35 @@ def process_tags(queryset):
     return ", ".join([tag for tag in queryset])
 
 
+def process_docket_numbers(queryset):
+    """Grab the one to many Docket Numbers values"""
+    return ",".join([d.docket_number for d in queryset])
+
+
+def process_content(page_content):
+    content = ""
+    soup = BeautifulSoup(str(page_content), "html.parser")
+    para = soup.findAll(["p", "h5"])
+    for p in para:
+        content += p.get_text()
+        link = p.find("a", href=True)
+        if link:
+            content += ": "
+            content += link["href"]
+        content += "\n"
+    return content
```

Comment:
I'm of two minds about this function — it could live as part of the model on a property too. But it's very specific to the report.

Because this file includes all our Wagtail reports, I think it'd be good to name it specific to the Enforcement Actions report, at least, and then we can leave it here:


```suggestion
def process_enforcement_action_page_content(page_content):
    content = ""
    soup = BeautifulSoup(str(page_content), "html.parser")
    para = soup.findAll(["p", "h5"])
    for p in para:
        content += p.get_text()
        link = p.find("a", href=True)
        if link:
            content += ": "
            content += link["href"]
        content += "\n"
    return content
```